### PR TITLE
feat(project): Reorder project

### PR DESCRIPTION
**Pull request by victor-carpio**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,15 +14,15 @@ const Button = styled.button`
   ${(props) =>
     props.primary &&
     css`
+      color: ${props.theme.colors.primary};
```
Comment (victor-carpio) [inline src/App.jsx:17]: This is just for testing theme provider

File: vite.config.js [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [reactRefresh(), eslintPlugin()],
+  resolve: {
+    alias: {
+      'assets': resolve(__dirname, 'public/assets/'),
+      'styles': resolve(__dirname, 'src/styles/'),
+      'components/': resolve(__dirname, 'src/components/'),
+      'utils/': resolve(__dirname, 'src/utils/'),
+      'actions/': resolve(__dirname, 'src/actions/'),
```
Comment (victor-carpio) [inline vite.config.js:15]: Just some examples. By the moment just using assets and styles

File: package.json [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
+  "name": "findme",
+  "version": "1.0.0",
   "scripts": {
     "dev": "vite",
     "build": "vite build",
     "serve": "vite preview",
     "lint": "eslint src/**.jsx"
   },
   "dependencies": {
-    "components": "^0.1.0",
```
Comment (victor-carpio) [inline package.json:10]: I think they were not really needed? 🤔

File: src/App.jsx [hunk 4]
```diff
@@ -14,15 +14,15 @@ const Button = styled.button`
...
       background: palevioletred;
-      color: white;
     `}
 `;
 const Container = styled.div`
   text-align: center;
 `;
 
-function App() {
+export default function App() {
```
Comment (lfernandezcall) [inline src/App.jsx]: Why not using arrow function here?

**Review by lfernandezcall — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 4 above)
Comment (victor-carpio) [reply src/App.jsx]: why not? We can define the use or function or arrow functions for components. I do not have any special preference between the use of `function` or `const`

Comment (victor-carpio) [reply src/App.jsx]: Let's use arrow functions for the project

**Pull request merged**